test/exprtest/CMakeLists.txt

```diff
@@ -26,13 +26,16 @@ cmake_minimum_required(VERSION 3.2.2)
 
 project(syclblas_exprtest)
 
+add_definitions(-D${TARGET})
+
 set(SYCLBLAS_EXPRTEST ${CMAKE_CURRENT_SOURCE_DIR})
 
 # compiling tests
 set(SYCL_EXPRTEST_SRCS
   ${SYCLBLAS_EXPRTEST}/blas1_scal_asum_test.cpp
   ${SYCLBLAS_EXPRTEST}/blas1_axpy_copy_test.cpp
   ${SYCLBLAS_EXPRTEST}/collapse_nested_tuple.cpp
+  ${SYCLBLAS_EXPRTEST}/extension_reduction_partial_rows_test.cpp
```

Comment:
wrap this in BLAS_EXTENSION macro as well

Reply:
See the answer above. A good change but will go in another MR